semeio/jobs/rft/gendata_rft.py

```diff
@@ -158,18 +183,19 @@ def run(
                 well=well, time=time
             )
 
-            # Write trajectory and associated data to three  files,
+            # Write trajectory and associated data to ASCII files
             # individual pr. well and time, this is for GEN_DATA to pick up.
-            _write_gen_data_files(
-                trajectory_df,
-                os.path.join(outputdirectory, "RFT_{}_{}".format(well, report_step)),
-            )
+            _write_gen_data_files(trajectory_df, outputdirectory, well, report_step)
 
             # Aggregate dataframe for all wells and report steps.
             dframes.append(trajectory_df)
+        else:
+            logger.warning(
+                "No trajectory points for well {} at date: {} found".format(well, time)
```

Comment:
Should be changed to `logger.error`? Because if this happens, the script will throw a `ValueError`

Reply:
Fixed.